docs/software/node.md

```diff
@@ -2,7 +2,7 @@
 
 ## Introduction
 
-The iris executable is the entry point for running a IRISnet network node. All the validator nodes and full nodes need to install the iris command and launching the daemon to join the IRISnet network. You can also use this command to start your own test network locally.
+The iris executable is the entry point for running a IRISnet network node. All the validator nodes and full nodes need to install the iris and launching the daemon to join the IRISnet network. You can also use iris to start your own test network locally.
 
 ## How to install IRIShub 
```

Comment:
How to install IRIShub --> How to install `iris`

Reply:
fixed